src/libraries/sendtohelixhelp.proj

```diff
@@ -157,6 +149,22 @@
     <HelixPreCommand Include="printenv | grep COMPlus" />
   </ItemGroup>
 
+  <!-- 
+    We are hosting the payloads for the WASM/browser on Kestrel in the xharness process.
+    We also run some network (HTTP and HTTPS) tests that connect to this server.
+    In order to set up an HTTPS endpoint we need to install a development SSL certificate.
+  -->
+  <PropertyGroup Condition="'$(TargetOS)' == 'Browser' and '$(HelixCommand)' == '' and '$(TargetsWindows)' != 'true' and '$(BrowserHost)' != 'windows'">
```

Comment:
This consolidation breaks the groupings for the properties that we have for the various cases, and makes the condition complicated. I would suggest:

1. add this
```xml
  <PropertyGroup Condition="'$(TargetOS)' == 'Browser' and '$(NeedsDotNetSdkWithCli)' == 'true'">
    <IncludeDotNetCli>true</IncludeDotNetCli>
    <DotNetCliPackageType>sdk</DotNetCliPackageType>
  </PropertyGroup>
```

2. and then set `NeedsDotNetSdkWithCli` wherever we have the `IncludeDotNetCli=true`, and `DotNetCliPackageType=sdk`.

That should retain the local grouping, and have consolidation around those properties.

Reply:
And maybe skipping the sdk for `Scenario=BuildWasmApps` will help with the failures in that.